config/translator.php

```diff
@@ -151,7 +151,7 @@
                  * To speed up detection, all the results of the scan will be stored in a file.
                  * Feel free to change the path if needed.
                  */
-                'cache_path' => base_path('.translator.cache'),
```

Comment:
Could you update the readme to ignore the right path in .gitignore?

Reply:
updated